tests/integration/test_bundler.py

```diff
@@ -0,0 +1,75 @@
+import logging
+from pathlib import Path
+
+import pytest
+
+from . import utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "test_params",
+    [
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachi2-bundler.git",
+                ref="ad1778cf87f45838330d63743a13fc33daa4f376",
```

Comment:
Why did you choose to directly refer to a commit hash instead of using branch name? The [relevant part](https://github.com/containerbuildsystem/cachi2/blob/main/tests/integration/utils.py#L173) of utils seems to just check out the reference. This should work with branch names too. If someone discovers an issue in test repo they would need to not just fix it, but to also update these reference.

Reply:
:thinking: that's a good point; we use commit hash everywhere, so that's why I did likewise.
(There are a lot of things within integration tests that should be refactored)

Reply:
I would replace the e2e test for **well_formed_ruby_without_gemspec** for a multi-package test, but it depends on how we resolve conversations in #654.

https://github.com/cachito-testing/cachi2-bundler/tree/main